src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -339,12 +342,15 @@ subroutine step_MOM_dyn_split_RK2(u, v, h, tv, visc, Time_local, dt, forces, p_s
     hf_CAu_2d, hf_CAv_2d, & ! Depth integeral of hf_CAu, hf_CAv [L T-2 ~> m s-2].
     hf_u_BT_accel_2d, hf_v_BT_accel_2d ! Depth integeral of hf_u_BT_accel, hf_v_BT_accel
 
+  real, allocatable, dimension(:,:,:) :: h_PFu, h_CAu, h_u_BT_accel ! [L2 T-2 ~> m2 s-2].
```

Comment:
The units in the comments for these diagnostics are incorrect.  They should be [H L T-2 ~> m2 s-2 or kg m-1 s-2]

Reply:
Thanks. I have made the corrections.